### PR TITLE
Fixing package versions

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* For details see Eric's comments for each commit.
* These changes means we will be building and running tests against a post release/1.0.0 version of CoreFx, TFS and CoreCLR packages.
* A couple fixes to the package validation section in dir.props and a version update to baseline.props will build correct WCF packages via the VSO pipeline that currently have some incorrect package dependency versions.
* Once this goes in and we have new WCF packages built we can update the version of the WCF packages.

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
Thanks @ericstj for putting this together.

**Comment by StephenBonikowsky** [comment]
@dotnet-bot test all outerloop please

File: Packaging.props [hunk 1]
```diff
@@ -4,11 +4,6 @@
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
     <PackageLicenseFile>$(ProjectDir)pkg/dotnet_library_license.txt</PackageLicenseFile>
     <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
-    
-    <!-- *** start WCF Content *** -->
-    <!-- *** this file is located differently in corefx *** -->
-    <RuntimeIdGraphDefinitionFile>$(PackagesDir)Microsoft.NETCore.Platforms/1.0.1-rc3-24206-00/runtime.json</RuntimeIdGraphDefinitionFile>
-    <!-- *** end WCF Content *** -->
```
Comment (iamjasonp) [inline Packaging.props]: Per Eric's commit and @StephenBonikowsky - this is now in buildtools, so this no longer needs to appear in our Packaging.props

**Comment by iamjasonp** [comment]
LGTM pending CI passes

**Comment by StephenBonikowsky** [comment]
@dotnet-bot test outerloop Ubuntu14.04

**Pull request merged**